README.md

```diff
@@ -21,7 +21,8 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+that is the migration file to create a table titled "artists"
+the numbers represent the migration id. This allows you to refer to that migration at any time and tracks the progress of our table
```

Comment:
close! the numbers are actually a timestamp, that tell rails what order to run the files in